### ISSUE TITLE
Feature request: option to disable pass-through of input messages

### ISSUE THREAD

**Issue by bartbutenaers** [Sineos/node-red-contrib-timeouttrigger#4]
Hi @Sineos,

Thanks for sharing this useful node!!

I was wondering if you could add an option to disable the passthrough of input messages to the output?  Currently both the input messages and the timeout messages will arrive on the output.  However my flow would be much simpler if only the timeout messages would be send on the output.

For example by adding a checkbox *"Disable pass-through of input messages"* on the node's config screen.

Kind regards,
Bart

**Comment by Sineos** [comment]
Hi @bartbutenaers. Seems trivial enough, even for my skills. Will take a look

**Comment by Sineos** [comment]
Added in v0.0.3

**Comment by Nexus1212** [comment]
Hi,
the problem still exists in V0.0.3!

![grafik](https://user-images.githubusercontent.com/25401458/227563010-fa50df0e-80e2-42bf-b4e0-e53be3499bd8.png)

**Comment by Sineos** [comment]
<del>Seems logical since you did not check "Pass-through messages"</del>
Ah, sorry, let me check

**Comment by Sineos** [comment]
Should be fixed in 0.0.4. Feedback would be appreciated

**Comment by bartbutenaers** [comment]
@Sineos,

OMG it seems I have never got back to you.  For some reason I have completely overlooked your response.  My sincere apologies for that!   Shame on me...

I have now taken the liberty to refresh your node on flows.nodered.org:

![image](https://user-images.githubusercontent.com/14224149/227624124-9555f959-20d8-4ab0-a970-79b2cffbdda2.png)

Since the automatic refresh has been removed quite some time ago, the version will not be updated if you don't trigger it manually...

I have tried a couple of scenario's and it seems to be working fine!  So thanks to this update, I can simplify my flow after all...

Thanks for your update!!!
Bart
